gridt/src/app/home/expandable/expandable.component.scss

```diff
@@ -0,0 +1,17 @@
+.p {
```

Comment:
I need a computer to be able to check out the functionality of this. But i won't be home until Thursday. Maybe you could make me a gif so I can see it?

Reply:
yes, i will send you one soon.

Reply:
It is in the slack/client